src/test/java/cydeo/emrah/carBrand.txt

```diff
@@ -0,0 +1,3 @@
+Car brand
+Ford
+Audi
```

Comment:
@bltemrah 
I think you forgot "bmw " as I remember one of the AC says bmw should accept as one of the car brand

Reply:
@kayla0408 Ooo yes your're right, let me update/add it.